packages/builder/src/components/design/PropertiesPanel/DesignView.svelte

```diff
@@ -64,10 +69,20 @@
       {/if}
     </div>
   </div>
+  {#if componentDefinition?.transitionable}
+     <div class="transitions">
+       <Select value={componentInstance._transition || ""} on:change={event => onUpdateTransition(event.target.value)} name="transition" label="Transition" secondary thin>
+         <option value="">Choose a transition</option>
+         {#each transitions as transition}
+           <option value={transition}>{transition}</option>
```

Comment:
Might be nice to capitalise the transitions as labels, just to make it more readable.

Reply:
Hmm, thought I did add a text-transform in the css. I guess that doesn't work in option tags (which makes sense in hindsight). 🤔 Good catch!